test/python/visualization/test_circuit_text_drawer.py

```diff
@@ -5975,145 +3737,6 @@ def test_if_op_bundle_true(self):
-                "«                                                                     ║    »",
-                "«cr_1: ═══════════════════════════════════════════════════════════════╬════»",
-                "«                                                                     ║    »",
-                "«cr_2: ═══════════════════════════════════════════════════════════════■════»",
-                "«                                                                          »",
-                "«                               ───────┐      ",
-                "« q_2: ────────────────────────        ├──────",
-                "«      ┌───┐ ───────┐ ┌───────┐        │ ┌───┐",
-                "« q_0: ┤ X ├        ├─┤0      ├        ├─┤ X ├",
-                "«      └───┘        │ │       │  End-0 │ └───┘",
-                "« q_3: ─────  End-1 ├─┤       ├        ├──────",
-                "«      ┌───┐        │ │       │        │      ",
-                "« q_1: ┤ X ├        ├─┤1 Inst ├        ├──────",
-                "«      └───┘ ───────┘ │       │ ───────┘      ",
-                "«cr_0: ═══════════════╡0      ╞═══════════════",
-                "«                     │       │               ",
-                "«cr_1: ═══════════════╡1      ╞═══════════════",
-                "«                     └───────┘               ",
-                "«cr_2: ═══════════════════════════════════════",
-                "«                                             ",
-            ]
-        )
-        qr = QuantumRegister(4, "q")
-        cr = ClassicalRegister(3, "cr")
-        circuit = QuantumCircuit(qr, cr)
-
-        circuit.h(0)
-        with circuit.if_test((cr[1], 1)) as _else:
-            with self.assertWarns(DeprecationWarning):
-                circuit.x(0, label="X c_if").c_if(cr, 4)
-            with circuit.if_test((cr[2], 1)):
-                circuit.z(0)
-                circuit.y(1)
-                with circuit.if_test((cr[1], 1)):
-                    circuit.y(1)
-                    circuit.z(2)
-                    with circuit.if_test((cr[2], 1)):
-                        circuit.cx(0, 1)
-                        with circuit.if_test((cr[1], 1)):
-                            circuit.h(0)
-                            circuit.x(1)
-        with _else:
-            circuit.y(1)
-            with circuit.if_test((cr[2], 1)):
-                circuit.x(0)
-                circuit.x(1)
-            inst = QuantumCircuit(2, 2, name="Inst").to_instruction()
-            circuit.append(inst, [qr[0], qr[1]], [cr[0], cr[1]])
-        circuit.x(0)
-        self.assertEqual(
-            str(
-                circuit_drawer(
-                    circuit,
-                    output="text",
-                    fold=77,
-                    initial_state=False,
-                    wire_order=[2, 0, 3, 1, 4, 5, 6],
-                )
-            ),
-            expected,
-        )
-
```

Comment:
I think these tests are still necessary?

Reply:
Restored in: https://github.com/Qiskit/qiskit/pull/13506/commits/6f3861acbf80a115679ffa7f66f55a108b3c6af8